docs/develop/scheduler-policy.md

```
@@ -201,3 +201,92 @@ the existing scheduling behavior. When present, the annotation must specify
+extenders:
+  - urlPrefix: http://hami-scheduler.kube-system.svc
+    filterVerb: filter
+    bindVerb: bind
+    httpTimeout: 30s        # must be >= node-lock-retry-timeout (default 28s)
+    nodeCacheCapable: false
+    ignorable: false
+```
+
+And the corresponding scheduler flag:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)